### ISSUE TITLE
Support Eclipse Che using the language server

### ISSUE THREAD

**Issue by gaocegege** [coala/coala-vs-code#44]

**Comment by gaocegege** [comment]
Che is based on Docker and we could build a sidecar for the LS: https://www.eclipse.org/che/docs/6/che/docs/language-servers.html#ls-sidecars

**Comment by ksdme** [comment]
I don't think it is logical to do that in coala-vs-code's scope. I mean we should develop a neutral coala lang server and then support Che or any other editor using specific means.

**Comment by gaocegege** [comment]
Yeah, that is what we need. But we do not have such a repo for language server so this issue is placed in this repo.
